components/Sections/Expertise.jsx

```diff
@@ -1,15 +1,15 @@
 import expertiseList from "../../utils/expertiseData";
 export default function Expertise() {
   return (
-    <section className="float-right">
+    <section className="float-right mb-[4.8125rem] md:mb-[8.25rem] lg:mb-24">
```

Comment:
It would be better if you add them in tailwind config

Reply:
I mean the arbitrary values

Reply:
Okay..